config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_for :users
+  resources :groups do
+    resources :payments
+  end
```

Comment:
- [OPTIONAL] It would be nice to use only, except, ... options to add only necessary routes in case maybe there are some routes that are not used. 👍🏻

Reply:
Hi @BenMukebo, I implemented all actions and routes in my controller, I'm planning to add options to delete or edit categories and payments.